scripts/setup_for_wrf.py

```diff
@@ -16,102 +16,59 @@
 import stat
 import netCDF4
 from setup_runs.wrf.fetch_fnl import download_gdas_fnl_data
+from setup_runs.config_read_functions import (read_config_file,
+                                              parse_config,
+                                              add_environment_variables,
+                                              substitute_variables,
+                                              parse_boolean_keys,
+                                              process_date_string,
+                                              load_wrf_config)
 
 ## get command line arguments
 parser = argparse.ArgumentParser()
 
-parser.add_argument("-c", "--configFile", help="Path to configuration file", default = 'config.nci.json')
+# TODO: change default file path back to `config.nci.json`
+parser.add_argument("-c", "--configFile", help="Path to configuration file", default = '../config.nci.json')
```

Comment:
In pycharm you should set the project root as the default working directory. A lot of these scripts assume that is the case. You are looking to edit the Python configuration template. There is a button in the bottom right of the edit run configuration pop up.

Reply:
The aim is to make the default to run:
`python scripts/setup_for_wrf.json`

instead of

`cd scripts; python setup_for_wrf.json` which is the default